staging/src/k8s.io/kubectl/pkg/cmd/get/sorter_test.go

```diff
@@ -435,6 +440,308 @@ func TestSortingPrinter(t *testing.T) {
 			},
 			field: "{.lastTimestamp}",
 		},
+		{
+			name: "pod-resources-cpu-random-order-with-missing-fields",
+			obj: &corev1.PodList{
```

Comment:
Just a NIT: As an effort to maybe make this tests more 'readable' (and also less lines of codes!) I'm wondering if maybe it isn't a good idea to instead of keep repeating the 

```
Spec: corev1.PodSpec [....]
```

we couldn't create a helper function here, like createPodSpecResource(memreq, memlim, cpureq, cpulim).This would return the correct spec, and could be called on every part of the code that uses this. We can reduce 11 lines per corev1.Pod spec here (I've counted 242 lines), and would add...maybe 30 lines on the helper function? :)

Reply:
Great call!